docs/source/samples/StereoDepth/stereo_depth_video.rst

```diff
@@ -11,6 +11,14 @@ inside the code:
 #. `extended`: suitable for short range objects. For more information `click here <https://docs.luxonis.com/en/latest/pages/faq/#extended-disparity-depth-mode>`__
 #. `subpixel`: suitable for long range. For more information `click here <https://docs.luxonis.com/en/latest/pages/faq/#subpixel-disparity-depth-mode>`__
 
+Stereo Alpha Param
+##################
+
+With `--alpha` you can select how much cropping will be done when rectifying the image, which is useful with Wide FOV OAK cameras. By default it's 0.0, which is cropping the most, so all points have valid disparity.
```

Comment:
It's not cropping per se, but rather scaling. I would like to take this opportunity to refer to the documentation from OpenCV for getOptimalNewCameraMatrix:

Free scaling parameter between 0 (when all the pixels in the undistorted image are valid) and 1 (when all the source image pixels are retained in the undistorted image). For details, see [stereoRectify](https://docs.opencv.org/4.9.0/d9/d0c/group__calib3d.html#ga617b1685d4059c6040827800e72ad2b6).

It's crucial to highlight its main use: increasing the depth of field (FOV). The theoretical range for this parameter is [0,1]. However, one can use values like -0.1 or -0.2 to eliminate the invalid image areas introduced by rectification rotation. This effect is achieved by "zooming into the image".

Reply:
Will update